index.html

```diff
@@ -4,19 +4,12 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./src/App.css">
```

Comment:
You don't need to do this. Just import css file in App.jsx and it will be automatically bundled as needed.